### DISCUSSION TITLE
Pickit Location

### DISCUSSION THREAD

**Discussion by ebner20** [isid0re/SoloLeveling#69] [Ideas]
Quick question, in what script does it point to your pickits?  I see they are commented out in the character configs so I figured you have some code pointing it at a custom one.  

I just cleaned up an assassin pickit and was going to try it out.  I made it for leveling and equipping  class specific (or caster specific) gear and the Merc.  

I've also finished up yesterday 3 builds for the Barbarian.  ShoutTank, BOBarb and Whirlwind.  I'm going to work on a frenzy barb eventually but wanted to test the first 3 before I dive into the 4th.  I can create another thread for Barb specifically.  I'm working on a pickit  specifically for the barb tonight but can't seem to find where you link it at and I don't want to add mine and break the bot.  

I was thinking of adding something like this in the build file itself for each type of build...(maybe its a little ocd of me)
for the assassin builds it would be
var AutoBuildTemplate = {
			SoloLevelingPickit: ["SoloLeveling/caster.xpac.nip", "SoloLeveling/assassin.trapsin.xpac.nip", "SoloLeveling/merc.xpac.nip"]

**Comment by isid0re** [comment]
I dont use pickits for autoequip, in the traditional sense.

I have a variable at the bottom of the config called, "autoequiptiers" 

Where the contents consist of 2 lines per  item type.  
1 is the finalbuild gear. Which is tier set to 100000
And the second uses [tier] == tierscore(item) 

Tierscore is a custom rating function based on if the build is a castor or melee.  Also it looks at wanted skills and useful skills when rating. 

I've been testing some major sequence edits for improving run times so I haven't had a chance to go thru your posts yet. Since I've got 4 chars running with an A|B test I will check them out tomorrow. And help clean them up for functionality.
